site/docs/hosting/comparison.md

```diff
@@ -29,19 +29,20 @@ We therefore recommend you to use a different kind of hosting for more complex b
+| ---------------------- | ---------- | --------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------- | --------------------------- | --- | ------------------------------ |
+| Deta                   | Free       | No paid plans yet                                                     | No specific limits                                                                      | ✅       | ❓                           | ❓   |                                |
+| Deno Deploy            | Free       | $10/mo subscription for 5M req and 100 GB; $2/1M req, $0.3/GB network | [100K req/day, 100 GB/mo, 10 ms CPU-time limit](https://deno.com/deploy/pricing)        | ❌       | ✅                           | ❌   |                                |
+| DigitalOcean Functions | Free       | $1.85/100K GB-s                                                       | [90K GB-s/mo](https://docs.digitalocean.com/products/functions/details/pricing/)        | ✅       | ❌                           | ❓   |                                |
+| Cloudflare Workers     | Free       | $5/10M req                                                            | [100K req/day, 10 ms CPU-time limit](https://workers.cloudflare.com/)                   | ❌       | [✅](https://denoflare.dev/) | ✅   |                                |
+| Heroku                 | Free       | It's complicated                                                      | [550-1000 h/mo](https://www.heroku.com/pricing)                                         | ✅       | ❓                           | ❓   |                                |
+| Vercel                 | Free       | $20/mo subscription                                                   | [Unlimited invocations, 100 GB-h, 10 s time limit](https://vercel.com/pricing)          | ❓       | ❓                           | ❓   | Not intended for non-websites? |
+| Scaleway Functions     | Free       | €0.15/1M req, €1.2/100K GB-s                                          | [1M requests, 400K GB-s/mo](https://www.scaleway.com/en/pricing/#serverless-functions)  | ❓       | ❓                           | ❓   |                                |
+| Scaleway Containers    | Free       | €0.10/100K GB-s, €1.0/100K vCPU-s                                     | [400K GB-s, 200K vCPU-s/mo](https://www.scaleway.com/en/pricing/#serverless-containers) | ❓       | ❓                           | ❓   |                                |
+| Vercel Edge Functions  | Free       | $20/mo subscription for 500K                                          | [100K req/day](https://vercel.com/pricing)                                              | ❓       | ❓                           | ❓   |                                |
```

Comment:
> `$20/mo subscription for 500K`

What does the `500k` mean here? The number of requests? I couldn't find any information on their website either.

Reply:
Scroll down.

Reply:
Oh wait, I thought you were asking about the `K`, never mind

Reply:
Let's ask @Loskir

Reply:
There used to be info about the number of requests (and 500k meant 500k/day for $20/mo), but they removed it somewhen around July (https://web.archive.org/web/20220623153752/https://vercel.com/pricing)

Reply:
How would you suggest to update the comparison?

Reply:
i'm not sure how their serverless stuff works (and if it's even possible to run grammy there), so then let's remove this info and leave a "not tested" comment

Reply:
I wonder if their current "edge middleware" thing is the same as "edge functions". If so, then it says "1 million invocations" under free and "1m + $0.65/m after" under $20/mo

Reply:
which is I suppose means 1m per month. which is 33k per day. which is not as worthwhile as it sounds